client/commonFramework/detect-unsafe-code-stream.js

```diff
@@ -5,9 +5,10 @@ window.common = (function(global) {
   } = global;
 
   const detectFunctionCall = /function\s*?\(|function\s+\w+\s*?\(/gi;
+  const detectFunctionInComment = /(\/\/)[a-z0-9\s]*?function|(\/\*)[a-z0-9\s\W]*?function[a-z0-9\s\W]*?(\*\/)/gi;
```

Comment:
This is breaking the CI.
You need to break this to lesser than 80 chars, as multiline
Refer: http://stackoverflow.com/questions/12317049/how-to-split-a-long-regular-expression-into-multiple-lines-in-javascript